pkg/apis/controller/experiments/v1beta1/experiment_types.go

```diff
@@ -207,6 +207,9 @@ type TrialTemplate struct {
 
 	// List of parameters that are used in trial template
 	TrialParameters []TrialParameterSpec `json:"trialParameters,omitempty"`
+
+	// Labels that determines if pod needs to be injected by Katib sidecar container
+	PrimaryPodLabels map[string]string `json:"primaryPodLabels,omitempty"`
```

Comment:
So for TFJob, user has to explicitly add primary labels ?

Reply:
User should specify it only in Experiment.
In TFJob for master pod, tfjob-controller automatically adds master labels.

As I said before we can set default values for `PrimaryPodLabels` for Kubeflow operators (TFJob, MPIJob).

Reply:
SGTM